tasks/agent.py

```diff
@@ -237,7 +237,9 @@ def image_build(ctx, base_dir="omnibus", python_version=2, skip_tests=False):
 
     # Pull base image with content trust enabled
     pull_base_images(ctx, "Dockerfiles/agent/Dockerfile", signed_pull=True)
-    common_build_opts = "-t {} --build-arg PYTHON_VERSION={}".format(AGENT_TAG, python_version)
+    common_build_opts = "-t {}".format(AGENT_TAG)
+    if python_version:
```

Comment:
With `python_version` expected to be an integer due to its default value, the idea here is to call the invoke task with `--python-version=0`, right? It doesn't look very explicit to me. Maybe time to make it a string and check if it's in `[ "2", "3", "both" ]` or something similar?